deoxys/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "deoxys"
+version = "0.1.0"
+authors = ["RustCrypto Developers, zer0x64"]
+edition = "2018"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+documentation = "https://docs.rs/deoxys"
+repository = "https://github.com/RustCrypto/AEADs"
+keywords = ["aead", "deoxys", "deoxys-i", "deoxys-ii"]
+categories = ["cryptography", "no-std"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+aead = { version = "0.4", default-features = false }
+subtle = "2.4.0"
+zeroize = { version = "1", default-features = false }
+
+[target.'cfg(target_feature="aes")'.dependencies]
+aes = { git = "https://github.com/RustCrypto/block-ciphers", rev="5e020f14d5fbff531bb6f9f15e9b97c366b8919c", features=["hazmat"]}
```

Comment:
FYI, `aes` v0.7.3 is out with this feature/API.

Still no software fallback support, but I'll look at that when I get some time (or if anyone else would like to take a look, by all means, but it will be a bit tricky)

Reply:
I've updated the import!